### ISSUE TITLE
Uncaught server error: java.lang.NoClassDefFoundError: org/keycloak/services/resources/Cors on Keycloak 25.0.0

### ISSUE THREAD

**Issue by laopaoer-wallet** [p2-inc/keycloak-events#61]
[org.keycloak.services.error.KeycloakErrorHandler] (executor-thread-44) Uncaught server error: java.lang.NoClassDefFoundError: org/keycloak/services/resources/Cors
        at io.phasetwo.keycloak.resources.CorsResource.setupCors(CorsResource.java:37)
        at io.phasetwo.keycloak.resources.AbstractAdminResource.setupCors(AbstractAdminResource.java:61)
        at io.phasetwo.keycloak.resources.AbstractAdminResource.setup(AbstractAdminResource.java:48)
        at io.phasetwo.keycloak.resources.WebhooksResourceProvider.getRealmResource(WebhooksResourceProvider.java:14)
        at io.phasetwo.keycloak.resources.BaseRealmResourceProvider.getResource(BaseRealmResourceProvider.java:30)
        at org.keycloak.services.resources.RealmsResource.resolveRealmExtension(RealmsResource.java:269)
        at org.keycloak.services.resources.RealmsResource$quarkusrestinvoker$resolveRealmExtension_c7e400748748bf35309bc60d82a4789f2a0e2d02.invoke(Unknown Source)
        at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
        at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
        at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
        at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:582)
        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
        at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
        at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.ClassNotFoundException: org.keycloak.services.resources.Cors

**Comment by laopaoer-wallet** [comment]
![image](https://github.com/p2-inc/keycloak-events/assets/75466368/522bd813-ce56-4416-8d24-e14223d3944e)

**Comment by xgp** [comment]
This is not yet ported to Keycloak 25. 

https://github.com/p2-inc/keycloak-events/pull/60

**Comment by 2tvenom** [comment]
hello

#60 is merged and i checked it on Keycloak 25.0.1 in docker (tag v0.30)

still got this error

```
2024-07-04 13:49:32,791 ERROR [org.keycloak.services.error.KeycloakErrorHandler] (executor-thread-9) Uncaught server error: java.lang.NoClassDefFoundError: org/keycloak/services/resources/Cors
	at io.phasetwo.keycloak.resources.CorsResource.setupCors(CorsResource.java:37)
	at io.phasetwo.keycloak.resources.AbstractAdminResource.setupCors(AbstractAdminResource.java:61)
	at io.phasetwo.keycloak.resources.AbstractAdminResource.setup(AbstractAdminResource.java:48)
	at io.phasetwo.keycloak.resources.WebhooksResourceProvider.getRealmResource(WebhooksResourceProvider.java:14)
	at io.phasetwo.keycloak.resources.BaseRealmResourceProvider.getResource(BaseRealmResourceProvider.java:30)
	at org.keycloak.services.resources.RealmsResource.resolveRealmExtension(RealmsResource.java:269)
	at org.keycloak.services.resources.RealmsResource$quarkusrestinvoker$resolveRealmExtension_c7e400748748bf35309bc60d82a4789f2a0e2d02.invoke(Unknown Source)
	at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:582)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.ClassNotFoundException: org.keycloak.services.resources.Cors
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at io.quarkus.bootstrap.runner.RunnerClassLoader.loadClass(RunnerClassLoader.java:115)
	at io.quarkus.bootstrap.runner.RunnerClassLoader.loadClass(RunnerClassLoader.java:65)
	... 17 more
```

**Comment by xgp** [comment]
fixed by #60
